docs/CheckpointerSchema.md

```diff
@@ -0,0 +1,169 @@
+Pact Database ("Checkpointer") Schema
+==========
+
+Schema documentation for Checkpointer Pact DB implementation in SQLite.
+
+
+Checkpointer System Tables
+===
+
+## BlockHistory table
+
+Tracks the latest block (by hash) and ending `TxId` for a given block height.
```

Comment:
By "ending `TxId`" - do we mean the last `TxId` in the block?

Reply:
It's clarified below. I'll add some intro text to indicate the doc structure.